util/feature.txt

```diff
@@ -0,0 +1,6 @@
+feature 1 implement
+
+feature 2 implement
```

Comment:
let's add a bit more to the feature2 implementation. I think it should handle multiple weights

Reply:
@meme2k21 see the latest commit for the implementation.